engine/controllers/models.cc

```diff
@@ -282,7 +292,8 @@ void Models::GetModel(const HttpRequestPtr& req,
       auto resp = cortex_utils::CreateCortexHttpTextAsJsonResponse(ret);
       resp->setStatusCode(drogon::k200OK);
       callback(resp);
-    } else {
+    } else if (model_config.engine == kOpenAiEngine ||
```

Comment:
If we go down this path, I think we should check engine == python here, rather than check engine == any remote engine. Because remote engine will have a lot more in the future (mistral, etc.)

so, I think in else if case, should be kPython, leave the remote engine to else as before